examples/scripts/gkd.py

```diff
@@ -93,6 +93,7 @@
 
     tokenizer = AutoTokenizer.from_pretrained(
         model_config.model_name_or_path,
+        revision=model_config.model_revision,
```

Comment:
Fix for cases where you need to load from a revision

Reply:
Good point, I'll extend this to all examples